doc/CODING.md

```diff
@@ -96,7 +96,12 @@ with `ks`, `key` or `kdb`.
 
 ### Doxygen Guidelines ###
 
-Use `doxygen` to document APIs, if available.
+`doxygen` is used to document the API, if available. We support also the import
```

Comment:
"if available" is not from you but its wrong. Either fix to "is used to build" or remove "is available"

witch -> which

avoid camelcase for link (MarkdownLinkConverter, folder is correctly named markdownlinkconverter)